test/euclidean.jl

```diff
@@ -2,14 +2,16 @@ include("utils.jl")
 
 @testset "Euclidean" begin
     E = Manifolds.Euclidean(3)
+    Ec = Manifolds.Euclidean(3;field=ℂ)
     EM = Manifolds.MetricManifold(E,Manifolds.EuclideanMetric())
     @test is_default_metric(EM) == Val{true}()
     @test is_default_metric(E,Manifolds.EuclideanMetric()) == Val{true}()
     x = zeros(3)
     @test det_local_metric(EM,x) == one(eltype(x))
     @test log_local_metric_density(EM,x) == zero(eltype(x))
     @test project_point!(E,x) == x
-
+    @test manifold_dimension(Ec) == 2*manifold_dimension(E)
```

Comment:
I think we should run full tests on `Ec`.

Reply:
No problem, I can do a few numbers for that manifold (though Euclidean really has a lot of tests already currently).

Reply:
Calling `test_manifold` should be enough I think.